pkg/util/file.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// isDirectory checks if a path points to a directory
+func isDirectory(path string) (bool, error) {
```

Comment:
We don't have any code like this or in or dependencies?

Reply:
I've just taken a look at the util package in kubernetes but i can't anything that stands out .. I'll try again though